components/eam/bld/namelist_files/use_cases/scam_generic.xml

```diff
@@ -6,4 +6,10 @@
 <ncdata             >atm/cam/inic/homme/cami_mam3_Linoz_ne4np4_L72_c160909.nc</ncdata>
 <soag_ext_file      >atm/cam/chem/trop_mozart_aero/emis/aces4bgc_nvsoa_soag_elev_2000_c160427.nc</soag_ext_file>
 
+<!-- SCM is configured with ne4 grid, but we want settings for an ne30 simulation -->
+<nu dyn_target="theta-l" hgrid="ne4np4"  >  1.0e15    </nu>
```

Comment:
To help my understanding: Is nu actually used anywhere in a SCM simulation?

Reply:
good find, yeah this is not needed and will be removed